docs/proposals/compute-resource-metrics-api.md

```diff
@@ -115,13 +116,11 @@ only be available through the kubelet. The types of metrics are detailed
 Additionally, all endpoints (except root discovery endpoint) support the
 following optional query parameters:
 
-- `start` - start time to return metrics from; type json encoded
-  `time.Time`; since samples are retrieved at discrete intervals, the first
-  sample after the start time is the actual beginning.
-- `end` - end time to return metrics to; type json encoded `time.Time`
-- `step` - the time step between each stats sample; type int (seconds), default
```

Comment:
Another question: Should the most recent metric always be included, and then "step" backwards from there? Or should the step be from some fixed time in the past (e.g. epoch)?

For example,
Step from most recent sample:
```
Input: start:=100 end=201 step=30
==> Output: samples[200, 170, 140, 110]
Input start=100 end=211 step=30
==> Output: samples[210, 180, 150, 120]
```
vs. Step from fixed time (0 in this example):
```
Input: start:=100 end=201 step=30
==> Output: samples[180, 150, 120]
Input start=100 end=211 step=30
==> Output: samples[210, 180, 150, 120]
```